pkg/flowaggregator/flowaggregator.go

```diff
@@ -591,3 +605,48 @@ func (fa *flowAggregator) fillK8sMetadata(key ipfixintermediate.FlowKey, record
 		}
 	}
 }
+
+func (fa *flowAggregator) fetchPodLabels(podAddress string) string {
+	pods, err := fa.podInformer.Informer().GetIndexer().ByIndex(podInfoIndex, podAddress)
```

Comment:
Just want to confirm if this works for dual-stack clusters where two IP addresses are there. Could you run dual-stack v6 e2e tests?

Reply:
Thanks, both v6 and dual stack e2e tests have passed.